test/validators.js

```diff
@@ -6222,6 +6221,7 @@ describe('Validators', () => {
           '015412345678',
         ],
         invalid: [
+          '+4930405044550',
```

Comment:
this is landline all +4930 are landlines. Mobile phone numbers must start with https://en.wikipedia.org/wiki/Telephone_numbers_in_Germany#Non-geographic_numbering must start with a 15, 16 or 17